src/Controllers/AdminController.php

```
@@ -697,11 +697,12 @@ public function billing(): void
         $activeAccounts = count(array_filter($activeUsers, static fn($managedUser) => (bool) ($managedUser['is_active'] ?? false)));
         $activeCount = User::countByOrgId($this->db, $orgId);
         $availableSeats = max(0, $seatLimit - $activeCount);
+        $fetchLiveStripeDashboardData = $this->shouldFetchLiveStripeDashboardData();
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $fetchLiveStripeDashboardData. Keep variable name length under 20. (undefined)

(LongVariable)